README.md

```diff
@@ -0,0 +1,85 @@
+| Linpack                                            | Licensed version of Linpack benchmark (see also: HPL)                               | https://github.com/redhat-performance/linpack-wrapper      |
+| NUMA STREAM                                        | Extension of STREAM benchmark to measure NUMA effects                               | https://github.com/redhat-performance/numa_streams-wrapper |
+| Passmark                                           | CPU benchmark                                                                       | https://github.com/redhat-performance/passmark-wrapper     |
+| pig                                                | Processor scheduler efficiency test                                                 | https://github.com/redhat-performance/pig_wrapper          |
+| Python Performance Benchmark Suite (pyperformance) | Performance testing for Python using real-world benchmarks                          | https://github.com/redhat-performance/pyperf-wrapper       |
+| SPEC CPU 2017                                      | Suite of benchmarks for compute-intensive performance                               | https://github.com/redhat-performance/speccpu2017-wrapper  |
+| SPEC JBB2005                                       | Evaluate the performance of server side Java                                        | https://github.com/redhat-performance/specjbb-wrapper      |
+| STREAM                                             | Benchmark for sustained memory bandwidth                                            | https://github.com/redhat-performance/streams-wrapper      |
+| Uperf                                              | Network performance testing                                                         | https://github.com/redhat-performance/uperf-wrapper        |
+
+## How It Works
+1. Zathras is installed to a controller system.
+2. Zathras runs using either a configuration file or by supplying parameters in your command-line interface (CLI) (see the [CLI reference](docs/command_line_reference.md) for more info).  
+3. Test(s) run on the system under test (SUT).
+4. When tests finish, Zathras copies the results back to the controller system.
+5. Once results have been retrieved, Zathras will attempt to cleanup the testing environment (such as by tearing down instances in the cloud or by removing directories and files on bare metal). 
+
+The user configures the Zathras installation with information about the system under test (SUT) and what test(s) to run.
+
+## Getting Started
+
+### Requirements
+To use Zathras, you need: 
+1. A system that will coordinate the execution of tests, which we will call the *control node* or simply the *controller*.
+2. One or more systems where tests will be run. This can be another physical system (aka bare metal), an instance on a public cloud like Amazon Web Services (AWS), or even a virtual machine (VM) either on the controller or another host system.
```

Comment:
For cloud.  Instance is created for us.  If the cloud instance already exists we can treat it as a bare metal machine.